schemas/EGA.experiment.json

```diff
@@ -2,12 +2,324 @@
     "$schema": "https://json-schema.org/draft/2019-09/schema",
     "$id": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.experiment.json",
     "type": "object",
-    "title": "EGA experiment metadata schema",
+    "title": "EGA Experiment metadata schema",
     "meta:version": "0.0.0",
-    "description": "Empty for the moment.",
-    "required": [""],
+    "description": "Metadata schema used by the European Genome-phenome Archive (EGA) to validate its experiment metadata object. An experiment is considered a planned and intentionally designed process performed as part of a study. Further details can be found in the EGA-metadata-schema GitHub repository (https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas) and EGA-archive website (https://ega-archive.org/)",
+    "required": ["object_id", "sample_number", "experiment_protocols", "assay_technology", "experimental_designs", "assay_type_descriptor", "assayed_molecule", "experiment_type_specifications"],
```

Comment:
```suggestion
    "required": ["object_id", "experiment_protocols", "assay_technology", "experimental_designs", "assay_type_descriptor", "assayed_molecule", "experiment_type_specifications"],
```

Reply:
Wondering if it's confusing that some of the attributes in the experiment object are named `assay_*`, although they are not in the assay object. Perhaps all attributes in experiment object should start with `experiment_*`.

Reply:
I don't have strong opinions on neither options, since I agree that having experiment_* is more straightforward, but having assay_technology aligns better with the attribute's definition and meaning in the ontology.